.genignore

```diff
@@ -4,7 +4,7 @@
 src/unstructured_client/_unstructured
 _jupyter
 _sample_docs
-_test_unstructured_client
```

Comment:
What was the reason for the file name change? I'm open to changing it, but I found the convention of using a leading underscore helpful for organizing the human-written code that we maintain.

Reply:
done, ready to resolve